pkg/services/ngalert/sender/router.go

```diff
@@ -75,20 +75,19 @@ func NewAlertsRouter(multiOrgNotifier *notifier.MultiOrgAlertmanager, store stor
 // SyncAndApplyConfigFromDatabase looks for the admin configuration in the database
 // and adjusts the sender(s) and alert handling mechanism accordingly.
 func (d *AlertsRouter) SyncAndApplyConfigFromDatabase() error {
-	d.logger.Debug("start of admin configuration sync")
 	cfgs, err := d.adminConfigStore.GetAdminConfigurations()
 	if err != nil {
 		return err
 	}
 
-	d.logger.Debug("found admin configurations", "count", len(cfgs))
```

Comment:
Let's keep it debug. This is a quite hot path and it will create a lot of noise. The same as in https://github.com/grafana/grafana/pull/57299